### PR TITLE
deploy: Fixes to deploy in toolsbeta

### PR DESCRIPTION
**Pull request by david-caro**
This adds some of the missing bits found when trying to deploy in toolsbeta

Bug: T297090

### CODE REVIEW

**Comment by david-caro** [comment]
This has been deployed to toolsbeta as is, ready to review.

**Review by vivian-rook — COMMENTED** [review]
Would
deploy/toolforge/kustomization.yaml and deploy/toolsbeta/kustomization.yaml be better as a symlink to the same file or are the expected to diverge?

**Comment by david-caro** [comment]
> Would deploy/toolforge/kustomization.yaml and deploy/toolsbeta/kustomization.yaml be better as a symlink to the same file or are the expected to diverge?

I expect them to diverge yes, specially the tag and the certificate should end up diverging (have not yet deployed it to tools, that's why they are currently the same xd)

File: README.md [hunk 1]
```diff
@@ -48,13 +48,27 @@ Since this was designed for use in [Toolforge](https://wikitech.wikimedia.org/wi
 The version of docker on the builder host is very old, so the builder/scratch pattern in
 the Dockerfile won't work.
 
-* Build the container on the docker-builder host (currently tools-docker-imagebuilder-01.tools.eqiad1.wikimedia.cloud). `root@tools-docker-imagebuilder-01:~# docker build . -f Dockerfile -t docker-registry.tools.wmflabs.org/buildpack-admission:latest`
-* Push the image to the internal repo: `root@tools-docker-imagebuilder-01:~# docker push docker-registry.tools.wmflabs.org/buildpack-admission:latest`
-* The caBundle should be set correctly in a [kustomize](https://kustomize.io/) folder. You should now just be able to run `root@tools-k8s-control-1:# kubectl apply -k deploy/toolforge` to deploy to tools and `root@toolsbeta-test-k8s-control-1:# kubectl apply -k deploy/toolsbeta` to make the deployment work.
+* Build the container on the docker-builder host (currently tools-docker-imagebuilder-01.tools.eqiad1.wikimedia.cloud).
```
Comment (aborrero) [inline README.md:53]: Perhaps you could leave a reference somewhere to [T291915](https://phabricator.wikimedia.org/T291915) for future reference.

**Comment by aborrero** [comment]
+1

File: README.md (hunk 1 above)
Comment (david-caro) [reply README.md:53]: :+1:

**Pull request merged**